src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/ResearchSetup.java

```diff
@@ -271,6 +271,9 @@ public static void setupResearches() {
         register("lead_clothing", 260, "Lead Clothing", 14, SlimefunItems.REINFORCED_CLOTH);
         register("tape_measure", 261, "Tape Measure", 7, SlimefunItems.TAPE_MEASURE);
         register("iron_golem_assembler", 262, "Automated Iron Golems", 30, SlimefunItems.IRON_GOLEM_ASSEMBLER);
+        register("unemployment_rune", 263, "Reset Villager Trades", 29, SlimefunItems.UNEMPLOYMENT_RUNE);
+        register("organic_nether_goo", 264, "Organic Nether Goo", 14,SlimefunItems.ORGANIC_NETHER_GOO);
```

Comment:
I don't really see why we would need a new research for this.

Reply:
Didn’t you suggest that I should add the research on the last set of changes?

“ dniym  Hide conversation
  @TheBusyBiscuit
TheBusyBiscuit yesterday  Owner
Your Nether Goo should probably also be added to this research”

Reply:
As I said...

Your Nether Goo should probably also **be added to this research**.

This research, not a new one.

Reply:
got it